app/configurations/config.tampere.js

```diff
@@ -38,12 +38,10 @@ export default configMerger(walttiConfig, {
   feedIds: ['tampere'],
 
   realTime: {
-    /*
     tampere: {
       gtfsRt: 'http://data.itsfactory.fi/journeys/api/1/gtfs-rt/vehicle-positions',
-      routeSelector: 'shortName',
+      routeSelector: (routePageProps) => routePageProps.route['shortName'],
```

Comment:
This will not work in IE11 - at least previously no transpiling was done for the configuration files. I would suggest using the function notation.